packages/@adobe/spectrum-css-temp/components/slider/index.css

```diff
@@ -66,10 +66,10 @@ governing permissions and limitations under the License.
   z-index: auto;
 
   min-block-size: var(--spectrum-slider-height);
-  min-inline-size: var(--spectrum-slider-min-width);
+  min-inline-size: var(--spectrum-global-dimension-size-2400);
```

Comment:
Should I create a PR to adjust the min-width to 192px here? https://git.corp.adobe.com/Spectrum/spectrum-dna/blob/main/data/elements/slider/SliderBase.mjs#L40

Reply:
Perhaps a variable in the `:root` of this file would suffice for now? Something like `--spectrum-slider-min-height: var(--spectrum-global-dimension-size-2400);`

Reply:
Ah never mind, my mistake. Just realized that `--spectrum-slider-min-width` is already a thing

Reply:
https://git.corp.adobe.com/Spectrum/spectrum-dna/pull/534

Reply:
Is it supposed to change the min width or just the default width?

Reply:
min width is default width, right?